src/main/java/frc/robot/subsystems/intake/BallIntake.kt

```diff
@@ -0,0 +1,63 @@
+package frc.robot.subsystems.intake
+
+import edu.wpi.first.math.controller.ArmFeedforward
+import edu.wpi.first.math.geometry.Rotation2d
+import edu.wpi.first.math.trajectory.TrapezoidProfile
+import edu.wpi.first.wpilibj2.command.Subsystem
+import frc.robot.utils.PIDCoefficients
+import frc.robot.utils.PIDController
+import frc.robot.CANDevice
+import org.littletonrobotics.junction.Logger
+import kotlin.math.PI
+import kotlin.math.max
+
+object BallIntake : Subsystem {
+
+    val pidController = PIDController(PIDCoefficients(0.0,0.0,0.0))
```

Comment:
You can just omit the arguments to `PIDCoefficients` if they're zero:
```suggestion
    val pidController = PIDController(PIDCoefficients())
```